bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -69,12 +69,13 @@ class CheckRun {
     private static final String fullNameWarningMarker = "<!-- PullRequestBot full name warning comment -->";
     private final static Set<String> primaryTypes = Set.of("Bug", "New Feature", "Enhancement", "Task", "Sub-task");
     private final Set<String> newLabels;
+    private final boolean requiresReviewForBackport;
```

Comment:
Can we change the name and the configuration option to `reviewCleanBackport`

Reply:
Sure!